src/main/java/frc/robot/subsystems/shooter/ShooterIOInputs.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.epilogue.Logged;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+
+/**
+ * The inputs used by the shooter subsystem
+ */
+@Logged
+public class ShooterIOInputs {
+
+        public double talon_state;
+        public double talon_velocity;
+        public double victor_output;
+
+        public Value elevationSolenoid_state;
```

Comment:
Instead of using DoubleSolenoid.Value for this input, please use a boolean. The reason we do this is that it makes logging easier to read and keeps the Subsystem logic easy and clean.